Source/MoreDotNet.Test/Extensions/Common/StreamExtensions/ToByteArrayTests.cs

```diff
@@ -0,0 +1,50 @@
+﻿namespace MoreDotNet.Tests.Extensions.Common.StreamExtensions
+{
+    using System;
+    using System.IO;
+    using MoreDotNet.Extensions.Common;
+    using Xunit;
+
+    public class ToByteArrayTests
+    {
+        [Fact]
+        public void ToByteArray_ThrowsOnNull()
```

Comment:
Please use the flowing naming convention for tests:

[MethodName]_[TestSetup]_Should[ExpectedResult]

Example for this:
ToByteArray_OnNullGiven_ShouldThrowException